include/scene.h

```diff
@@ -21,6 +21,10 @@ class Scene {
   const std::vector<std::unique_ptr<SceneObject>>& get_objects() const {
     return scene_objects_;
   }
+
+  const std::vector<std::unique_ptr<SceneObject>>& get_lights() const {
```

Comment:
naaaah.. I actually think SceneObject should be replaced with Light in that line..